setup.py

```diff
@@ -303,8 +303,8 @@ def check_package_status(package, min_version):
         },
     ],
     "neighbors": [
-        {"sources": ["_ball_tree.pyx"], "include_np": True},
-        {"sources": ["_kd_tree.pyx"], "include_np": True},
+        {"sources": ["_ball_tree.pyx.tp", "_binary_tree.pxi.tp"], "include_np": True},
+        {"sources": ["_kd_tree.pyx.tp"], "include_np": True},
```

Comment:
Is it possible to decouple those two lines:

```suggestion
        {"sources": ["_binary_tree.pxi.tp"], "include_np": True},
        {"sources": ["_ball_tree.pyx.tp"], "include_np": True},
        {"sources": ["_kd_tree.pyx.tp"], "include_np": True},
```
?

Reply:
How about using this then?
```python
        {"sources": ["_ball_tree.pyx.tp", "_binary_tree.pxi.tp"], "include_np": True},
        {"sources": ["_kd_tree.pyx.tp", "_binary_tree.pxi.tp"], "include_np": True},
```

Reply:
Won't this generate the file 2 times?